Apply oracle handler to convert unicode input to byte strings within DBCore code; revert patch for RESTModel

vkuznet opened the pull request.

vkuznet committed:
Apply oracle handler to convert unicode input to byte strings within DBCore code; revert patch for RESTModel

ericvaandering commented:
Looks like this blows up MySQL

vkuznet commented:
Now, once I see the error I'll fix it.

On  0, Eric Vaandering <notifications@github.com> wrote:
> Looks like this blows up MySQL
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6912#issuecomment-224368196

ericvaandering commented:
Much better.

amaltaro commented on src/python/WMCore/Database/DBCore.py:
```diff
@@ -134,6 +147,8 @@ def processData(self, sqlstmt, binds={}, conn=None,
                 connection = self.connection()
             else:
                 connection = conn
+            if  isinstance(self.engine.dialect, OracleDialect_cx_oracle):
```
Valentin, I know you like leaving two white spaces in an `if` clause, unfortunately it generates one more complaint line in the pylint output. You just told me you'll probably not have time today, thus please avoid them in future PRs.

amaltaro commented:
@vkuznet @ericvaandering I was about to miss this one, I believe we do want to have it in for tomorrow's tag, don't we?

amaltaro commented:
test this please

amaltaro commented:
And before I forget, @vkuznet please squash these commits :)

vkuznet commented:
Alan, I removed white space and squashed.
(extra white spaces in if statements look nice for code readability and you may consider
adjusting pylint settings).

On  0, Alan Malta Rodrigues <notifications@github.com> wrote:
> And before I forget, @vkuznet please squash these commits :)
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6912#issuecomment-228721555

vkuznet commented:
Yes we need to include this to deal with CherryPy + DBS.

On  0, Alan Malta Rodrigues <notifications@github.com> wrote:
> @vkuznet @ericvaandering I was about to miss this one, I believe we do want to have it in for tomorrow's tag, don't we?
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6912#issuecomment-228720305

amaltaro commented:
test this please

amaltaro commented:
Ok, the functionality seems to be Ok since I didn't see anything falling apart. We shall see how it really behaves in cmsweb :) Thanks Valentin
